src/xapiwrapper.js

```diff
@@ -165,8 +164,7 @@ function isDate(date) {
 
         function getbase(url)
         {
-            var l = document.createElement("a");
-            l.href = url;
+            var l = parseUrl(url);
```

Comment:
The `document` object doesn't exist in node so I've added a cross platform `parseUrl` function to appropriately differentiate.